### ISSUE TITLE
[classic] Hunter Pets Disappearing

### ISSUE THREAD

**Issue by scotty0100** [cmangos/issues#282]
I've noticed a lot of bug reports on my server in regards to Hunter Pets disappearing when they shouldn't. This is using the current core also.

It's hard to replicate but probably a bug that should be noted and perhaps looked at.

Generally what happens is that they log off, (seems to be more commonly in cities?), when the pet has good loyalty etc. And the next time they log in they are gone.

Any ideas why this could be? I've had 5-6 separate bug reports regarding this now so I don't think it's just due to low loyalty.

**Comment by cala** [comment]
@scotty0100 do you still encounter this issue?

**Comment by scotty0100** [comment]
Nope - when I upgraded the server and re installed everything on there, it went away.

Went from MYSQL 5.1.61 - Windows XP Machine 32bit
to 
MYSQL 5.6.17 Windows Server 2012 R2 64bit 

Not sure what exactly the root cause of this was, just happened on my old machine and not on the new one. So anyone else with this problem I would recommend probably re installing MYSQL from scratch and probably a more up to date version if its an older one. I seemed to be one of the only people that encountered this anyway!
